src/main/java/com/appleaster/command/Command.java

```diff
@@ -0,0 +1,56 @@
+package com.appleaster.command;
+import java.time.LocalDate;
+
+import com.appleaster.task.Task;
+
+public class Command {
+    private final CommandType type;
+    private Task task;
```

Comment:
The following fields could be effectively final as well, I noticed the rest of your code in this class do not further modify them after being initialised by the use of overloaded constructors